app/views/layouts/application.html.erb

```diff
@@ -32,6 +32,8 @@
   <% end %>
 <% end %>
 
+<%= image_tag("hpi_main_building_keyvisual_1020x420.jpg", :alt => "HPI Background", :class => 'banner') %>
```

Comment:
layouts/application means on any page there is :P you probably meant to put this in views/application/index